### DISCUSSION TITLE
Impressão p/ executante

### DISCUSSION THREAD

**Discussion by edu2005edu** [RamonSilva20/mapos#2124] [Ideas]
Gostaria de ver a possibilidade de implementação uma funcionalidade para muito importante na aplicação. Que é uma condição de impressão para o **tecnico executante** (sem valores expressos) apenas com o descritivo sem o valor. Essa condição apenas para quando vou imprimir a OS ( em caixa seletora,  ou conceder ou não esta função em um usuário). 
Pois o valor do orçamento não gostaria de disponibilizar para o executante do serviço, apenas a descrição do serviço que será realizado e os materiais, porém tudo sem o valor da OS para o tecnico ou executante levar para realizar o serviço. è possivel?

**Comment by almron** [comment]
Também acho esta uma função importante.

**Comment by KelvinEger** [comment]
Uma nova opção de impressão simples resolveria? 
Ex.: Uma nova opção de impressão sem os valores.

**Comment by edu2005edu** [comment]
olá, sim será muito bom. isso já resolve sim.

Em qua., 18 de out. de 2023 às 16:28, Kelvin Eger ***@***.***>
escreveu:

**Comment by edu2005edu** [comment]
Olá , agradeço a sua atenção, sim a função de simples impressão sem os
valores ajuda muito. Pois quando o executante/colaborador for prestar o
serviço o valor não pode estar disponível.

Em qua, 18 de out de 2023 16:28, Kelvin Eger ***@***.***>
escreveu:

**Reply by KelvinEger** [reply]
Beleza então.
Vou implementar, assim que possível estarei disponibilizando!
Vocês já utilizam o Mapos no dia-a-dia?

**Comment by edu2005edu** [comment]
Kelvin, boa tarde.
Agradeço por me responder , e por atender a demanda que precisamos.

Sim estamos usando sim a aplicação, vc está de parabéns.

Aguardo então a implantação.

E parabéns pelo App.

Em seg, 23 de out de 2023 15:08, Kelvin Eger ***@***.***>
escreveu:

**Comment by edu2005edu** [comment]
Ola Kevin, boa tarde, tenho atualizado o App, e cada dia está  melhor.
Gostaria de saber se vc aplicou a melhoria de Impressão  sem os Valores?

Att. Carlos Eduardo.

Em ter., 24 de out. de 2023, 14:51, C.Eduardo ***@***.***>
escreveu:

**Comment by KelvinEger** [comment]
Boa tarde, tudo bem??
Acabei migrando e focando no desenvolvimento de emissão de NF-e pelo
software.

Em seg., 28 de out. de 2024 às 14:30, edu2005edu ***@***.***>
escreveu:

**Comment by edu2005edu** [comment]
Kevin, boa noite agradeço muito seu retorno, o seu trabalho esta muito bom,
quanto a essa função estou esperançoso de que a aplicação  tenha, pois
preciso enviar para rua para realização  da OS, e o valor na OS esta me
atrapalhando muito.

Como vc disse, criar apenas a impressa sem o valor me ajuda bastante.


No mais muito obrigado.


Em seg., 28 de out. de 2024, 14:51, Kelvin Eger ***@***.***>
escreveu:

**Comment by edu2005edu** [comment]
Oi Kevin, boa tarde, tudo bem?

Caríssimo,  estou aqui aguardando dia a dia para a implantação desse
recurso no sistema. Precisamos muito emitir a OS , sem a descrição dos
valores, então  como mensionei anterirmente, qualquer recurso que vc
consiga criar no software, para que em alguns casos eu consiga fazer a
impressão  da OS sem o campo de valores será  ótimo.

Em seg., 28 de out. de 2024, 14:51, Kelvin Eger ***@***.***>
escreveu:

**Comment by maycon1977** [comment]
Acredito que eu consiga fazer uma versão em breve de impressão de O. S.
dessas. Compartilho sem problema, porém não sei se vai servir no seu
sistema, pois o meu tem algumas modificações.

Em qui., 16 de jan. de 2025, 14:20, edu2005edu ***@***.***>
escreveu:

**Reply by edu2005edu** [reply]
Olá, bom dia Mycon, gostaria de saber se vc conseguiu  fazer no sistema essa função de fazer uma impressão de os sem os valores expostos. Seria uma impressão para o tecnico de campo ir executar, porém não estar disponibilizado os valores aoplicados.

**Comment by edu2005edu** [comment]
Na realidade kevin, completando as informações,  poderia ser uma impressão
com os valores zerados, tanto os campos Produtos e Serviços, e o campo de
QRcode sem serem exibidos o valor, estamos usando a aplicação dia a dia,
porem não  posso imprimir, porque o colaborador não é para ter o acesso a
esta informação. Entao uma impressão onde os valores são  zerados ou nao
apresentados, entende.

Outra coisa na aba cadastro de clientes/Fornecedores temos o campo CNPJ que
ao ser inserido ja busca na receita o Nome Social. Todavia não tem um campo
com o nome FANTASIA, PORÉM  ESTE CAMPO PRECISA SER ADICIONADO MANUALMENTE,
sem ser buscado buscado na Receita federal. Isso facilita a busca pois tem
muitos clientes que usam o mesmo CNPJ, porem o Nome Fantasia é diferente, e
nao consigo localizar no cadastro. Pois pega apenas o que está no principal.


Essas alterações  são muito importantes, para o bom funcionamento e vai ser
positivo para todos.


Agradeço muito a atenção.



Em seg., 28 de out. de 2024, 14:51, Kelvin Eger ***@***.***>
escreveu:

**Reply by andremp15** [reply]
Mas não tem muito sentido essa funcionalidade de acrescentar mais uma opção, exemplo uma pessoa que MEI, ele tem tanto o nome quanto o nome da empresa fantasia com o mesmo título, outra observação como ficaria a parte de buscar esse cliente pessoa jurídica ao emitir uma ordem de serviço vai ser pesquisado pelo nome fantasia ou o pelo nome do cliente eu acho que, eu atualmente uso o nome fantasia abreviado com o nome do cliente para mim não tem muito relevância, porque na hora de imprimir a nota fiscal vai puxar sempre o nome fantasia, desde que não seja MEI,

**Comment by edu2005edu** [comment]
olá bom dia kevin, tudo bem? ontem fiz uma instalação da ultima versão aqui na minha maquina, para ver se algumas modificações que eu precisava foram feitas, e até o momento não. em um tempo atraz vc até disse que faria essas alterações. tem algum posicionamento a respeito?
hoje vejo que tem dois tipos de impressão, uma em A4 ( completa) e outra formato cupom, poderia ser mais um tipo porém sem valores na os.

 poderia ser uma impressão
com os valores zerados, tanto os campos Produtos e Serviços, e o campo de
QRcode sem serem exibidos o valor,

Outra coisa na aba cadastro de clientes/Fornecedores temos o campo CNPJ que
ao ser inserido ja busca na receita o Nome Social. Todavia não tem um campo
com o nome FANTASIA, PORÉM  ESTE CAMPO PRECISA SER ADICIONADO MANUALMENTE,
sem ser buscado buscado na Receita federal. Isso facilita a busca pois tem
muitos clientes que usam o mesmo CNPJ, porem o Nome Fantasia é diferente, e
nao consigo localizar no cadastro. Pois pega apenas o que está no principal. O exemplo de empresas é um Grupo ( que possui 1 cnpj mas tem varias franbquias dentro da empresa ( tipo praça de alimentação de shoping).

**Comment by andremp15** [comment]
Mas essa parte de visualização tanto da ordem de serviço com valor quanto na impressão isso é fácil de resolver é só você implementar em permissões do usuário, no meu eu fiz da seguinte maneira o usuário técnico funcionário ele não pode ver as aba de desconto produto e serviço isso já resolve. Soube imprimir lote de serviço sem valores por um usuário técnico funcionário eu não vejo muito sentido que deveria ter é assinatura digital do cliente.

**Comment by maycon1977** [comment]
Fiz sim, porém eu uso ainda uma versão antiga do sistema de 2018. Não
serviria para a versão atual do MAPOS.

*Atenciosamente,*
*Maycon Martins Teixeira ***@***.***_rio)*

*Telefone/**WhatsApp**: +55 (
<https://api.whatsapp.com/send?phone=5521988078360&text=Ol%C3%A1%2C%20Maycon.%20Sou%20um%20cliente%20da%20Press%C3%A3oRio.%20Pode%20me%20ajudar%3F>**21)
98807-8360
<https://api.whatsapp.com/send?phone=5521988078360&text=Olá%2C%20Maycon.%20Sou%20um%20cliente%20da%20PressãoRio.%20Pode%20me%20ajudar%3F>*


Em qui., 2 de abr. de 2026 às 10:26, edu2005edu ***@***.***>
escreveu:

**Comment by edu2005edu** [comment]
Boa tarde André, tudo bem?
meu caro, essas funções aqui para meu negócio são sim importantes para meu
dia a dia, como falei empresas de grupo usam o mesmo cnpj, porém com nome
fantasia diferente, quando vc vai em uma praça de alimentação de
shopping por exemplo, existem ali várias lojas da praça, meu exemplo aqui é
( pizzaria Alegria, Shoes, Divino fogão, Terra a Vista, Bom Burger, etc...
como vê várias franquias, com nomes diferentes, porém o cnpj é do Grupo
entende, e faço serviço para todas essas lojas, quando eu preciso buscar eu
só consigo um resultado. me dificultando encontrar qual é a loja que foi
feito o serviço entende.

Quanto ao outro email da impressão para mim também é importante, por causa
do meu ramo de atividade, eu preciso imprimir a os e meus técnicos vão para
campo executar, já tive muitos problemas com o valor exposto na os.

att Carlos Eduardo.

Em dom., 5 de abr. de 2026 às 08:50, AndrePereira ***@***.***>
escreveu:
